_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -182,6 +182,10 @@ OpenSearch supports the following dynamic index-level index settings:
 
 - `index.final_pipeline` (String): The final ingest node pipeline for the index. If the final pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name `_none` specifies that the index does not have an ingest pipeline.
 
+- `index.optimize_doc_id_lookup.fuzzy_set.enabled` (Boolean): This setting controls whether fuzzy set should be enabled for optimizing document id lookups in indexing/search calls by using an additional data structure (Bloom Filter). Enabling this improves performance for upsert and search operations which rely on document id by creating a new data structure (bloom filter) which allows to handle negative cases (i.e. ids being absent in the existing index) through faster off-heap look-ups. Default is `false`. This setting can only be used if the feature flag `opensearch.experimental.optimize_doc_id_lookup.fuzzy_set.enabled` is set to `true`.
```

Comment:
```suggestion
- `index.optimize_doc_id_lookup.fuzzy_set.enabled` (Boolean): This setting controls whether `fuzzy_set` should be enabled for optimizing document ID lookups in indexing or searching calls by using an additional data structure, in this case, the Bloom filter data structure. Enabling this setting improves performance for upsert and search operations that rely on document ID by creating a new data structure (Bloom filter). The document ID allows for the handling of negative cases (that is, IDs being absent in the existing index) through faster off-heap look-ups. Default is `false`. This setting can only be used if the feature flag `opensearch.experimental.optimize_doc_id_lookup.fuzzy_set.enabled` is set to `true`.
```

Reply:
Is it the document ID that allows for handling negative cases?

Reply:
@vagimeli Bloom filter allows for handling negative cases